swarm/tools/runs_gc.py

```
@@ -74,15 +75,25 @@ def age_days(self) -> float:
-        for entry in path.rglob("*"):
-            if entry.is_file():
-                try:
-                    total += entry.stat().st_size
-                except OSError:
-                    pass
-    except OSError:
-        pass
+
+    # PERFORMANCE OPTIMIZATION (Bolt): Replaced pathlib.Path.rglob with os.scandir
+    # for significantly faster recursive directory traversal when calculating total size.
+    def _scan(dir_path: str):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `_scan`

Add return type annotation: `None`

(ANN202)